docs/misc/math-expr.md

```diff
@@ -117,6 +117,12 @@ See javadoc of java.lang.Math for detailed explanation for each function.
 |acos|acos(x) would return the arc cosine of x|
 |asin|asin(x) would return the arc sine of x|
 |atan|atan(x) would return the arc tangent of x|
+|bitwiseAnd|bitwiseAnd(x,y) would return the result of x & y|
+|bitwiseComplement|bitwiseComplement(x) would return the result of ~x|
+|bitwiseOr|bitwiseOr(x,y) would return the result of x [PIPE] y |
```

Comment:
I added `[PIPE]` as I had trouble escaping `|` within the table. Can play with it though if necessary

Reply:
`&#124;` should work (it's done elsewhere in the file, for logical OR).